probe/probe.go

```diff
@@ -533,6 +534,11 @@ func printKubeArmorProbeOutput(buf []byte) error {
 	data = append(data, []string{" ", "Container Security:", green(strconv.FormatBool(kd.ContainerSecurity))})
 	data = append(data, []string{" ", "Container Default Posture:", green(kd.ContainerDefaultPosture.FileAction) + itwhite("(File)"), green(kd.ContainerDefaultPosture.FileAction) + itwhite("(Capabilities)"), green(kd.ContainerDefaultPosture.NetworkAction) + itwhite("(Network)")})
 	data = append(data, []string{" ", "Host Default Posture:", green(kd.HostDefaultPosture.FileAction) + itwhite("(File)"), green(kd.HostDefaultPosture.CapabilitiesAction) + itwhite("(Capabilities)"), green(kd.HostDefaultPosture.NetworkAction) + itwhite("(Network)")})
+	hv := strings.Split(kd.HostVisibility, ",")
```

Comment:
Are theese changes backwards compatible? Will new kArmor work with older KubeArmor where the visibility information is not available?

Reply:
With the proposed fix (comma separated HostVisibility), even with the absence of the attribute _HostVisibility_, the `karmor probe` does not fail, it just displays an empty string as below:

![image](https://user-images.githubusercontent.com/1323423/226471587-5e4cfc00-aabd-460f-9bbb-c7915ab4d3ed.png)